ParisBNF/abb/BNFabb106.xml

```diff
@@ -0,0 +1,234 @@
+                            <date>1902</date>.</acquisition>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:ContiRossini1914Notice"/>
+                                            <citedRange unit="page">120</citedRange>
+                                            <citedRange unit="number">88</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Chaine1912Catalogue"/>
+                                            <citedRange unit="page">66</citedRange>
+                                            <citedRange unit="number">106</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Abbadie1859Cat"/>
+                                            <citedRange unit="page">118-119</citedRange>
+                                            <citedRange unit="number">106</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:BmWebsite"/>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Chants"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Liturgy"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="NV" when="2024-09-09">I have complemented Conti Rossini's description with information provided by d'Abbadie and, based on the available images, described quire structure, ruling and stamp</change>
+            <change who="NV" when="2024-09-03">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                    <bibl>
```

Comment:
if this is bibliography relevant for the manuscript it should be in `listBibl type="secondary"`
if this is bibliography relevant for the text it must be under the relevant work, in `listBibl type="edition"` and `listBibl type="translation"` and not here at all

Reply:
I checked other mss desc and see that this element is never used, so I will not do it either, leaving only the note      <note> Used in the edition <bibl> <ptr target="bm:Velat1966Soma"></ptr></bibl>
                            </note>

Reply:
Once again: the bibliography for the CONTENT of a manuscript is the bibliography for the WORK and should go with the WORK record.

In the manuscript record we ONLY quote bibliography DESCRIBING THE MANUSCRIPT